TriblerGUI/widgets/trustpage/js/main.js

```diff
@@ -110,16 +124,27 @@ function handle_node_click(public_key) {
         console.log("Request pending, ignore new request");
     } else {
         if (public_key !== state.focus_pk) {
-            state.request_pending = true;
+            navigation.step(public_key);
+        }
+    }
+}
+
+function onStep(public_key, neighbor_level){
```

Comment:
This function feels better if he has proper documentation, so developers can read details about him.